contribs/gmf/src/directives/partials/editfeature.html

```diff
@@ -29,39 +29,80 @@
         </span>
       </span>
     </a>
-    <form
-      novalidate
-      name="form"
-      class="form gmf-editfeature-attributes-container"
-      ng-switch-default
-      ng-if="efCtrl.attributes">
-      <ngeo-attributes
-        ngeo-attributes-attributes="::efCtrl.attributes"
-        ngeo-attributes-disabled="efCtrl.pending"
-        ngeo-attributes-feature="::efCtrl.feature">
-      </ngeo-attributes>
-      <input
-        type="submit"
-        value="{{'Save' | translate}}"
-        class="btn btn-sm btn-default gmf-editfeature-btn-save"
-        ng-click="form.$valid && efCtrl.save()"
-        ng-disabled="!efCtrl.dirty"
-        title="{{'Save modifications | translate'}}"></input>
-      <a
-        class="btn btn-sm btn-default gmf-editfeature-btn-cancel"
-        ng-click="efCtrl.cancel()"
-        ng-disabled="efCtrl.pending"
-        title="{{'Cancel modifications | translate'}}"
-        href>{{'Cancel' | translate}}</a>
-      <button
-        class="btn btn-sm btn-link gmf-editfeature-btn-delete"
-        ng-click="efCtrl.delete()"
-        ng-disabled="efCtrl.pending"
-        ng-show="efCtrl.featureId"
-        title="{{'Delete this feature | translate'}}">
-        <span class="fa fa-trash"></span>
-        {{'Delete' | translate}}
-      </button>
-    </form>
+    <div ng-switch-default>
+      <ngeo-modal ng-model="efCtrl.unsavedModificationsModalShown">
```

Comment:
Putting the modal in the switch reduces the number of watchers.  It is also only required while a feature is selected, so putting it in the switch makes sense to me.  I also used the authentication directive as inspiration, which does that.  I was told that reducing the number of watchers is a priority in Angular, so this kind of wierd practice is a result of that.

What do you think ?